book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental)
```

Comment:
- [ ] You need to add the rentals to the attr_accessor for the getter and setter too. Inclusive, you need to initialize the array in the initialize method too.